Update readme (#17)
moneykit/examples: README.md

### Getting started

1. Copy the `.env` you created above to `create_link/.env`

   ```sh
   cp .env create_link/.env
   ```

2. Change directories into `create_link` and launch your preferred backend.

   ```sh
   cd create_link

   docker compose run --rm --service-ports backend-python
   # or

>    docker compose run --rm --service-ports backend-python-httpx

Reviewer: I don't have context on the reason for multiple python backends being here, but I think for the purposes of this section it might be simplest to just list one backend service so no one is confused about which one they should be using? (Probably the httpx - last time I tried to run this, that one worked and the other didn't)

**Web:**

1. In a separate shell, change directories to the frontend web project:

   ```sh
   cd create_link/frontend_web
   ```

2. Install dependencies and start the server.
   ```sh
   npm install
   npm run dev
   ```

> 3. At this point, the frontend should be running on port 3000 (in npm) and you should see "Local:  **http://localhost:3000**".

Reviewer: "(in npm)" does not really make sense here in the way it does for the Docker instructions above. I would just drop that part.

> 4. Click that to visit `http://localhost:3000` in your browser, and create a new link.

Reviewer: ```suggestion
4. Visit `http://localhost:3000` in your browser, and create a new link.
```

## Get some account data!

If you've followed the steps above, created a new link, and clicked Next all the way to the end, you should have been rewarded with a panel that shows a new `link_id`:

   > **link_id**<br>
   > mk_AfZGwnseamMJQcAWXqk7iL

You can now use Postman, curl, or any HTTP client to request accounts, balances, and other data about that link
using the MoneyKit API:  https://docs.moneykit.com.

   ```sh
   # create an access_token
   curl --request POST \
   --url 'https://production.moneykit.com/auth/token'
   --header 'accept: application/json'
   --header 'content-type: application/x-www-form-urlencoded'
   --data 'grant_type: client_credentials'
   --data 'client_id: your_client_id'
   --data 'client_secret: your_secret'

   # get accounts
   curl --request GET
   --url 'https://production.moneykit.com/links/{link_id}/accounts'
   --header 'accept: application/json'

>    --header 'Authorization: Bearer { your_auth_token }'

Reviewer: While I would hope the reader would figure out themselves that the braces here are just marking out the placeholder, I think this might be clearer without them, in case they are very unfamiliar with header syntax or curl etc.

```suggestion
   --header 'Authorization: Bearer YOUR_AUTH_TOKEN'
```

We also reference an "auth" token here vs. an "access" token in the comment above the first request above, we probably want to make those consistent.

   ```

> If you have Python and Poetry installed, you can also run a tiny demo app included in this repo:

Reviewer: Maybe make the purpose of this section slightly clearer. I.e. that it's an alternative to making the requests above manually.

```suggestion
A tiny demo app is also included which will take your link ID and request your accounts for you (requires Python and Poetry to be installed on your machine):
```

>    ```sh

Reviewer: Maybe specify which directory they should be in here for this part too.

Reviewer: For sure, I'm operating under the assumption that we fix what's broken here.
